packages/api/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "comments": false,
+  "sourceMaps": "both",
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "8.10"
+      }
+    }]
+  ],
+  "plugins": [
```

Comment:
Babel is necessary pretty much just to add this plugin, `source-map-support`. It turns out that in order for your error stack traces to actually reference the original source code,  in addition to generating source maps your code itself needs to register source map support. This babel plugin ensures that the compiled code will have those statements to register source map support:

https://www.npmjs.com/package/babel-plugin-source-map-support